docs/simplesamlphp-developer-information.md

```diff
@@ -98,13 +93,39 @@ The "docs" repo (as described in the readme of the repo) only contains
 the scripts that generate the docs website. In order to improve the
 content of the documentation themselves, you commit using the same branches used
 for code contributions at
-https://github.com/simplesamlphp/simplesamlphp.
+<https://github.com/simplesamlphp/simplesamlphp>.
 
 You can address documentation updates to master
-(https://github.com/simplesamlphp/simplesamlphp/tree/master/docs).
+<https://github.com/simplesamlphp/simplesamlphp/tree/master/docs>.
 Though it makes sense to backport them to supported releases, so each
-version under https://simplesamlphp.org/docs/<VERSION>/ will show the
+version under <https://simplesamlphp.org/docs/VERSION/> will show the
 change. In other words, if a documentation change relates to 2.1.3 you
 might like to make the pull request against the simplesamlphp-2.1
 branch and leave it to the team to also apply it to master and other
 branches in the same way that code updates work.
+
+### Documentation linting
+
+The CI system has some linting for markdown files in place. This uses
+`github-action-markdown-cli@v3` to perform the work which itself uses
```

Comment:
I would leave the version out, so we don't have to constantly update this text

Reply:
I have just removed that in the last push below :)